coffee-module/coffee-module-redisstream/src/main/java/hu/icellmobilsoft/coffee/module/redisstream/bootstrap/BaseRedisConsumerStarter.java

```diff
@@ -86,18 +95,43 @@
  */
 public class BaseRedisConsumerStarter {
 
+    private static final int DEFAULT_MAX_THREAD_COUNT = 16;
+    private static final String REDIS_PREFIX = "coffee.redis";
+
+    /**
+     * Managed executor service core thread count config string ({@value #MANAGED_EXECUTOR_SERVICE_CORE_THREADS})
+     */
+    protected static final String MANAGED_EXECUTOR_SERVICE_CORE_THREADS = "MANAGED_EXECUTOR_SERVICE_CORE_THREADS";
```

Comment:
Ez egy docker-wilfly base image specifikus env változó, ráadásul még azon belül is lehet `managed-executor-service.core-threads` property-t használni a wildfly.properties-en keresztül.
Jobb lenne ezt kiszervezni konfigurálhatóvá, meg akkor már a safety_buffer is jöhetne konfigból.